File: swagger/models/get_custom_payment_methods_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse

class GetCustomPaymentMethodsResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		payment_methods
		Contains information about the custom payment methods.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'PaymentMethods': 'payment_methods',
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'payment_methods': 'PaymentMethods',
		}

	pagination_response = None
	payment_methods = None

File: swagger/Models/GetCustomPaymentMethodsResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetCustomPaymentMethodsResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about the custom payment methods.
	 * 
	 * @var CustomPaymentMethod[] 
	 */
	public $paymentMethods;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'PaymentMethods' => ['paymentMethods', 'array', CustomPaymentMethod::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'paymentMethods' => ['PaymentMethods', 'array', CustomPaymentMethod::class],
			];
	}
}
